hs_geo_raster_resource/models.py

```diff
@@ -130,25 +131,29 @@ def add_to_xml_container(self, container):
 
         rdf_coverage_value.text = cov_value
 
-    def get_html_form(self, resource):
-        """Generates html form code for this metadata element so that this element can be edited"""
+    @classmethod
+    def get_html_form(cls, resource, element=None, allow_edit=True, file_type=False):
+        """Generates html form code for an instance of this metadata element so
+        that this element can be edited"""
 
         from .forms import OriginalCoverageSpatialForm
 
         ori_coverage_data_dict = dict()
```

Comment:
@pkdash better replace dict() with {} due to performance reasons. See http://stackoverflow.com/questions/664118/whats-the-difference-between-dict-and for details.